ansible/README_DISTRIBUTED.md

```diff
@@ -0,0 +1,110 @@
+Deploying Distributed OpenWhisk using Ansible
+=========
+
+### Distributed Deployment using Ansible
+**Caveat:** All Ansible commands are meant to be executed from the `ansible` directory.
+This is important because that's where `ansible.cfg` is located which contains generic settings that are needed for the remaining steps.
+
+By default, if you omit the `-i` parameter in ansible commands, the `local` environment will be used.
+
+In all instructions, replace `<openwhisk_home>` with the base directory of your OpenWhisk source tree. e.g. `openwhisk`
+
+Login to your bootsrapper VM. Your local machine can act as bootstrapper as well, if it can connect to other VMs.
```

Comment:
bootstrapper VM

can act as the bootstrapping machine as well, if it can connect to the VMs deployed in your IaaS.